tests/foreman/sys/test_fam.py

```diff
@@ -135,6 +135,10 @@ def test_positive_run_modules_and_roles(module_target_sat, setup_fam, ansible_mo
 
     :expectedresults: All modules and roles run successfully
     """
+    # Setup provisioning resources
+    if ansible_module in FAM_TEST_LIBVIRT_PLAYBOOKS:
+        module_target_sat.configure_libvirt_cr()
```

Comment:
This probably won't help as once one of the test runs `configure_libvirt_cr` then all the tests afterwards would be affected as they will be using `module_target_sat`. We probably need to separate the tests or use a different satellite for libvirt-related tests.

Reply:
But it's just the call to `conf_libvirt_cr()` that's failing, the overall `module_target_sat` won't be broken if the setup fails.

Reply:
ah okay